test/unit/htmlProcessing.test.js

```diff
@@ -87,6 +90,51 @@ describe('HTML processing', () => {
     })
   })
 
+  describe('inline code handling', () => {
+    describe('with inline code disabled', () => {
+      it('does not modify code block content', function () {
+        const codeBlock = this.output("pre:contains('codeblocks will not')")
+        assert.match(codeBlock.html(), /&lt;%-.*%&gt;/)
+      })
+
+      it('does not unescape delimited code', function () {
+        const className = this.output("p:contains('.purplePapyrus')")
+        const styleTag = className.prev()
+        const openingTag = styleTag.prev()
+
+        assert.equal(styleTag.html(), '&lt;style&gt;')
+        assert.equal(openingTag.html(), '&lt;%-')
+      })
+    })
+
+    describe('with inline code enabled', () => {
+      before(function () {
+        process.env.ALLOW_INLINE_CODE = 'true'
+        // remove formatter from require cache to recognize changed env variable
+        delete require.cache[require.resolve(formatterPath)]
```

Comment:
Not a fan of this and considered modifying the formatter functions to take `allowInlineCode` as an argument that defaults to the environment variable, but it seems like not a great idea to modify code to better fit our testing suite than to run the code as it would be in a production-like way...